pandas/core/sparse/frame.py

```diff
@@ -74,6 +75,8 @@ def __init__(
         dtype=None,
         copy=False,
     ):
```

Comment:
didn't we just merge this change?

Reply:
No, this is in #27302.

Reply:
do you want to move that first?

Reply:
This PR is rebased on top of that, on the assumption that will get merged before this

Reply:
ok